Sources/Logging/Logging.swift

```diff
@@ -39,20 +39,41 @@ import WASILibc
 /// logger.info("Hello World!")
 /// ```
 public struct Logger {
-    @usableFromInline
-    var handler: LogHandler
+    /// A private property to hold the boxed `LogHandler`.
+    private var _handler: Box<LogHandler>
 
     /// An identifier of the creator of this `Logger`.
     public let label: String
+    
+    /// A computed property to access the `LogHandler`.
+    public var handler: LogHandler {
+        get {
+            return _handler.value
+        }
+        set {
+            if !(isKnownUniquelyReferenced(&_handler)) {
```

Comment:
Should I use `isUniquelyReferenced` or `isUniquelyReferencedNonObjC`?

Reply:
`isKnownUniquelyReferenced` is correct